test/mocks/event/mocks.h

```diff
@@ -163,6 +174,28 @@ class MockTimer : public Timer {
   Event::TimerCb callback_;
 };
 
+class MockSchedulableCallback : public SchedulableCallback {
+public:
+  MockSchedulableCallback(MockDispatcher* dispatcher);
+
```

Comment:
nit: add destructor defintion and impl in cc also per gmock compile perf recommendations.